cdk/README.md

```diff
@@ -36,17 +36,28 @@ $ pip install -r requirements.txt
 
 You must now ensure that you are signed into AWS CLI:
 
-Instructions for configuring AWS CLI: 
-- https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html
+Instructions for configuring AWS CLI:
 
-Sign in instructions for AWS Educate accounts:
-- Navigate to awseducate.com and sign in
-- Go to "My Classrooms" and select go to classroom for the class' account you wish to deploy to
-- Select "Account Details" and "Show" next to AWS CLI
-- Copy and paste the block code that is shown into ~/.aws/credentials
+-   https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html
+
+In order to sign into your AWS account on the CLI, follow one of the 3 options below:
+
+    1. Configure the AWS CLI to use AWS SSO. This will allow you to use the AWS CLI without having
+    to manually navigate to the login page. To do this, follow the instructions in [this guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-sso.html#sso-using-profile). Keep in mind that if you go this route, in order
+    to get emporary credentials you will need ot run `aws sso login --profile your-profile-name` before each command.
+
+    2. Export the credentials to your environment variables. To do this, follow the instructions above to log in
```

Comment:
It doesn't look to me like there are any relevant instructions above? Can you double check this?

I see there used to be instructions in https://github.com/clemsonMakerspace/unified-makerspace/pull/53/commits/64985f7e38e3afdcd3b0c9747cefc8668c42b0ee but they were removed in https://github.com/clemsonMakerspace/unified-makerspace/pull/53/commits/825fd21936209b2d70ce83a72908d6635e58827c.